processing/src/main/java/org/apache/carbondata/processing/csvreaderstep/UnivocityCsvParser.java

```diff
@@ -41,6 +41,10 @@
 public class UnivocityCsvParser {
 
   /**
+   * Max number of columns that will be parsed for a row by univocity parsing
+   */
+  private static final int MAX_NUMBER_OF_COLUMNS_FOR_PARSING = 2000;
```

Comment:
I think user will not be able to provide this.He may get a csv with more cols, but he is interested in few cols he is interested in.Depending on user would be bug prone